htdocs/main.inc.php

```diff
@@ -2016,24 +2016,24 @@ function top_menu_user($hideloginname = 0, $urllogout = '')
 	$dropdownBody .= '<div id="topmenulogincompanyinfo" >';
 
 	if ($langs->transcountry("ProfId1", $mysoc->country_code) != '-') {
-		$dropdownBody .= '<br><b>'.$langs->transcountry("ProfId1", $mysoc->country_code).'</b>: <span>'.showValueWithClipboardCPButton($conf->global->MAIN_INFO_SIREN).'</span>';
+		$dropdownBody .= '<br><b>'.$langs->transcountry("ProfId1", $mysoc->country_code).'</b>: <span>'.showValueWithClipboardCPButton(!empty($conf->global->MAIN_INFO_SIREN)).'</span>';
```

Comment:
Instead of showing the value, you show true or false !
You can fix this by replacing 
$conf->global->XXXX with getDolGlobalString($conf->global->XXXX)